<launch>
  <node name="pc_visualizer" pkg="orchard_mapping" type="visualize_pc.py" />
  <node name="give_next_pc_server" pkg="ptcloud_visualization" type="give_next_pc_server.py" />
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find orchard_mapping)/rviz/pc_visualization.rviz" />
  <rosparam file="$(find orchard_mapping)/config/pt_params.yaml" command="load" />
  <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find package_name)/rviz/config_file.rviz" /> -->
</launch>
